include/odp/api/spec/crypto.h

```diff
@@ -143,13 +143,19 @@ typedef enum {
 	 */
 	ODP_AUTH_ALG_SHA256_HMAC,
 
+	 /** HMAC-SHA-384
+	 *
+	 *  SHA-384 algorithm in HMAC mode
+	 */
+	ODP_AUTH_ALG_SHA384_HMAC,
+
 	/** HMAC-SHA-512
 	 *
 	 *  SHA-512 algorithm in HMAC mode
 	 */
 	ODP_AUTH_ALG_SHA512_HMAC,
 
-	/** AES-GCM
+	/** AES in Galois/Counter Mode
```

Comment:
Should not change the heading for the algorithm. I'm using algorithm short names from specifications. E.g. RFC 4106 specifies the name like this.

"This document describes the use of AES in GCM mode (AES-GCM) as an
   IPsec ESP mechanism for confidentiality and data origin
   authentication.  We refer to this method as AES-GCM-ESP." 

And the section describing GCM is:
"2.  AES-GCM"

Reply:
Missed this change. Should be fixed now.